lib/storage/src/content_manager/consensus/persistent.rs

```diff
@@ -120,9 +120,8 @@ impl Persistent {
         self.apply_progress_queue.current()
     }
 
-    pub fn entry_applied(&mut self) -> Result<(), StorageError> {
+    pub fn entry_applied(&mut self) {
         self.apply_progress_queue.applied();
-        self.save()
```

Comment:
This single `save` call, when called after each entry application, slowed the whole consensus sync by up to 7.5 times...